src/main/java/edu/cornell/kfs/concur/ConcurKeyConstants.java

```diff
@@ -25,4 +25,11 @@
     public static final String CONCUR_SAE_NO_REPORT_EMAIL_SUBJECT = "message.concur.sae.no.report.email.subject";
     public static final String CONCUR_SAE_NO_REPORT_EMAIL_BODY = "message.concur.sae.no.report.email.body";
     public static final String CONCUR_INCOMPLETE_ADDRESS = "validation.error.concur.incomplete.address";
+    public static final String CONCUR_TOKEN_REQUEST_QUESTION = "message.concur.token.request.question";
```

Comment:
I realize now this file has not followed the general KFS convention of naming these constants starting with ERROR for errors MESSAGE for messages but perhaps we could start now?